### PR TITLE
Use `function_runner_with_pdb`, this will enter `pdb` when `--pdb` passed

### PR DESCRIPTION
**Pull request by myakove**
https://github.com/RedHatQE/qe-metrics/issues/80

### CODE REVIEW

**Comment by redhat-qe-bot2** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>


</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

File: qe_metrics/cli.py [hunk 1]
```diff
@@ -9,10 +8,34 @@
 from qe_metrics.libs.jira import Jira
 from qe_metrics.utils.issue_utils import create_update_issues
 from qe_metrics.utils.product_utils import products_from_file
+from pyhelper_utils.runners import function_runner_with_pdb
+
 
 LOGGER = get_logger(name="main-qe-metrics")
 
 
+@orm.db_session
```
Comment (myakove) [inline qe_metrics/cli.py:17]: @calebevans Do I need this on both functions?

Comment (calebevans) [reply qe_metrics/cli.py:17]: I think you only need it on this one (qe_metrics) @myakove

Comment (calebevans) [reply qe_metrics/cli.py:17]: If you setup a local config from the readme and run, it will throw an error if you need it.

Comment (myakove) [reply qe_metrics/cli.py]: I will check it

Comment (myakove) [reply qe_metrics/cli.py]: Seems to work without any

**Review by calebevans — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**